webui/react/src/pages/ExperimentList.tsx

```diff
@@ -308,7 +315,49 @@ const ExperimentList: React.FC = () => {
       value ? <Link path={paths.experimentDetails(value)}>{value}</Link> : null
     );
 
-    const tableColumns: ColumnsType<ExperimentItem> = [
+    const tableColumns: ExperimentColumnDefs = {
+      action:
+      {
+        align: 'right',
+        className: 'fullCell',
+        fixed: 'right',
+        key: 'action',
+        onCell: () => ({ isCellRightClickable: true } as React.HTMLAttributes<HTMLElement>),
+        render: actionRenderer,
+        title: '',
+        width: DEFAULT_COLUMN_WIDTHS['action' as ExperimentColumnName],
```

Comment:
chore: This causes weird behavior when you try to resize the action column.

Reply:
To be addressed in future UX refinement